src/app/(sidebar)/account/fund/page.tsx

```diff
@@ -104,10 +113,10 @@ export default function FundAccount() {
         </div>
       </Card>
 
-      {showAlert && isFetchedAfterMount && isSuccess && (
+      {showAlert && isFetchedAfterMount && isSuccess && account.publicKey && (
```

Comment:
We could create a Friendbot success and error message component to be shared between the Create and Fund pages. That way we would handle all the logic only once. What do you think?